poetry.lock

```diff
@@ -1005,58 +1351,67 @@ files = [
 
 [[package]]
 name = "numpy"
-version = "1.26.4"
+version = "2.0.0"
```

Comment:
FYI @zhindes If we were using Renovate for this repo, it would automatically run the PR build & tests when upstream dependencies are updated. (Or the CI build, if we configured it to bypass PR policy.)

As-is, we basically only update poetry.lock when someone changes something unrelated, like this PR.

Reply:
@bkeryan Should we keep this discussion open?

Reply:
No. I already fixed the test failures with NumPy 2.0, so this isn't breaking your PR build. I'll file a GitHub issue about moving to Renovate.

Reply:
Noted, I will close this conversation.